modules/jvm/src/test/scala/coursier/jvm/JvmCacheTests.scala

```diff
@@ -66,6 +74,14 @@ object JvmCacheTests extends TestSuite {
           |        "1.2": "tgz+https://foo.com/download/the-jdk-1.2.tar.gz"
           |      }
           |    }
+          |  },
+          |  "windows": {
+          |    "the-arch": {
+          |      "jdk@the-jdk": {
```

Comment:
I added some java dummy bat files in these archives, but maybe it's better to do these tests in terms of just reading the files content so it won't be platform dependent whether it'll run or not?

Reply:
It's fine this way I think. It's a good thing to actually run things on Windows.

Reply:
Agreed on that it's good to run things on windows. In the glorious future, that might be in separate tests, but at least we have something that can make noise when it stops running now.